Makefile

```diff
@@ -37,7 +37,9 @@ ifdef CLI_DIR
 	OC_CLI = ${CLI_DIR}/oc
 endif
 
+ifeq ($0,"")
 CLUSTER_TYPE ?= $(shell $(OC_CLI) get infrastructures cluster -o jsonpath='{.status.platform}' | awk '{print tolower($0)}')
```

Comment:
I think $0 here is run in a separate $(shell..) not the same shell as one that run `ifeq ($0,"")`